src/app/personside/dialogpanel/fortsettDialog/FortsettDialogContainer.tsx

```diff
@@ -73,40 +82,43 @@ function FortsettDialogContainer(props: Props) {
 
     const handleSubmit = (event: FormEvent) => {
         event.preventDefault();
-        if (isPosting(sendSvarResource)) {
+        if (isPosting(sendSvarResource) || !isFinishedPosting(opprettHenvendelseResource)) {
```

Comment:
Hva er forskjellen på de to metodene? Den er vel isPosting når det ikke er isFinishedPosting?

Reply:
den kan også være notStarted ;-) men den logikken her bør nok forbedres litt så den er mer lesbar